src/Helper/UserBookingsHelper.php

```diff
@@ -82,7 +82,10 @@ public function getResult(string $apiEndpoint, Request $request)
     if ($this->bookingApiEndpoint && $this->bookingApiKey) {
       switch ($apiEndpoint) {
         case "v1/user-bookings":
-          $response = $this->getData($apiEndpoint."?userId=1", $request->getQueryString());
+          if ($request->isMethod("DELETE")) {
+            $response = $this->deleteData($apiEndpoint, $request->getQueryString());
+          }
+          $response = $this->getData($apiEndpoint, $request->getQueryString());
```

Comment:
Should this be an else? 
In this implementation it will both call deleteData and getData if the method is DELETE...

Reply:
Whoops! Fixed